### PR TITLE
proxy support configure pulsar client prop: maxLookupRequest&concurrentLookupRequest

### PR DESCRIPTION
**Pull request by JoeCqupt**
proxy support configure pulsar client prop: maxLookupRequest&concurrentLookupRequest

#497

### CODE REVIEW

**Comment by Technoboy-** [comment]
Hi @JoeCqupt . Good work.
But the current implementation has some problems when publish in the proxy. We need to optimize first and then support this.

File: mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTCommonConfiguration.java [hunk 1]
```diff
@@ -168,6 +168,20 @@
...
+                    + "each broker connection to prevent overloading a broker."
+    )
+    private int mqttProxyClientMaxLookupRequest = 50000;
+
+    @FieldContext(
+            category = CATEGORY_MQTT_PROXY,
+            doc = "The number of concurrent lookup requests that can be sent on each broker connection. "
+                    + "Setting a maximum prevents overloading a broker."
+    )
+    private int mqttProxyClientConcurrentLookupRequest = 5000;
```
Comment (Technoboy-) [inline mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTCommonConfiguration.java:183]: It's better to move these two configs to `MQTTProxyConfiguration`.
And also rename to `maxLookupRequest`, `concurrentLookupRequest`

File: mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTProtocolHandler.java [hunk 2]
```diff
@@ -91,50 +91,13 @@ public void start(BrokerService brokerService) {
...
-
-            proxyConfig.setTlsTrustStore(mqttConfig.getTlsTrustStore());
-            proxyConfig.setTlsTrustCertsFilePath(mqttConfig.getTlsTrustCertsFilePath());
-            proxyConfig.setTlsTrustStoreType(mqttConfig.getTlsTrustStoreType());
-            proxyConfig.setTlsTrustStorePassword(mqttConfig.getTlsTrustStorePassword());
-
-            proxyConfig.setTlsEnabledWithKeyStore(mqttConfig.isTlsEnabledWithKeyStore());
-            proxyConfig.setTlsKeyStore(mqttConfig.getTlsKeyStore());
-            proxyConfig.setTlsKeyStoreType(mqttConfig.getTlsKeyStoreType());
-            proxyConfig.setTlsKeyStorePassword(mqttConfig.getTlsTrustStorePassword());
```
Comment (JoeCqupt) [inline mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTProtocolHandler.java:133]: the property not match, i think it maybe a mistake

File: mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTCommonConfiguration.java (hunk 1 above)
Comment (JoeCqupt) [reply mqtt-impl/src/main/java/io/streamnative/pulsar/handlers/mqtt/MQTTCommonConfiguration.java]: fine! i have updated

**Comment by JoeCqupt** [comment]
wip

**Comment by JoeCqupt** [comment]
PTAL @Technoboy-

**Comment by JoeCqupt** [comment]
#507 fix ci build

**Comment by mattisonchao** [comment]
Hi, @JoeCqupt 
#525 fixes the config issue, could you please recheck it and let this PR focus on adding config?

**Review by mattisonchao — APPROVED** [review]

**Comment by Technoboy-** [comment]
Thanks @JoeCqupt .

**Pull request merged**